packages/jest-cli/src/TestRunner.js

```diff
@@ -77,7 +76,7 @@ class TestRunner {
     this._setupReporters();
   }
 
-  addReporter(reporter: BaseReporter) {
+  addReporter(reporter: Object) {
```

Comment:
@thymikee I think this should be an Object as we are instantiating the reporters before calling addReporter and passing it the instance.